Sources/Swifter/HttpParser.swift

```diff
@@ -25,7 +26,9 @@ class HttpParser {
         if statusLineTokens.count < 3 {
             throw HttpParserError.InvalidStatusLine(statusLine)
         }
-        let method = statusLineTokens[0]
+        guard let method = HttpMethod(rawValue: statusLineTokens[0]) else {
```

Comment:
We might want to make this a little more forgiving, i.e. it could accept non-uppercase method strings from clients. What do you think?